### PR TITLE
BASE: Text Area

### PR DESCRIPTION
**Pull request by stasnowak**
#17 

Adding Text Area Base along with Syncfusion Blazor Nuget

textareabase

### CODE REVIEW

File: Taarafo.Portal.Web/Views/Bases/TextAreaBase.razor.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
```
Comment (hassanhabib) [inline Taarafo.Portal.Web/Views/Bases/TextAreaBase.razor.cs:1]: @stasnowak Add a header here.

**Comment by hassanhabib** [comment]
@stasnowak this is good work here. Just one comment

**Comment by stasnowak** [comment]
@hassanhabib resolved

**Comment by hassanhabib** [comment]
@stasnowak please resolve conflicts

**Pull request merged**